sources/packages/web/src/components/layouts/Institution/sidebar/HomeSideBar.vue

```diff
@@ -85,7 +91,17 @@ export default {
         });
       }
     };
-    onMounted(getuserLocationList);
+    watch(
+      () => userLocationList.value,
+      async () => {
+        // get user details
+        await getuserLocationList();
```

Comment:
It seems that this method doesn't need to be async anymore. Can you check, please?
Also, it seems that there is no reason to return it on the Setup method, can you check it also, please?